.platform.app.yaml

```diff
@@ -9,6 +9,11 @@ relationships:
     database: "mysql:mysql"
     redis: "redis:redis"
 
+
+variables:
+    env:
+        SYMFONY_ENV: 'prod'
```

Comment:
Are you sure it's `SYMFONY_ENV`, not `APP_ENV`?

Reply:
Yup Ref: https://github.com/Sylius/SyliusDemo/blob/master/bin/console#L20